{
    "inputs": {
        "dependency": "tensorflow",
        "outfile": "/home/vanessa/Desktop/Code/caliper-analysis/.caliper/pypi-tensorflow-2.0.2-python-cp36.json",
        "dockerfile": "FROM python:3.6\nADD . /tmp/repo\nWORKDIR /tmp/repo\nENV LANG C.UTF-8\nENV SHELL /bin/bash\nRUN apt-get update && apt-get install -y wget ca-certificates gnupg2 git\nRUN /bin/bash -c \"wget https://files.pythonhosted.org/packages/5f/a0/efe24ab4ea4a6ae6316cd421761978d8034fce144b3aa6e3ea342c8fbdc8/tensorflow-2.0.2-cp36-cp36m-manylinux2010_x86_64.whl && pip install tensorflow-2.0.2-cp36-cp36m-manylinux2010_x86_64.whl\"\n",
        "force": false,
        "exists": true,
        "name": "pypi-tensorflow-2.0.2-python-cp36",
        "tests": "tensorflow_v0.11/5_MultiGPU/multigpu_basics.py\ntensorflow_v0.11/1_Introduction/basic_operations.py\ntensorflow_v0.11/1_Introduction/helloworld.py\ntensorflow_v0.11/4_Utils/tensorboard_advanced.py\ntensorflow_v0.11/4_Utils/tensorboard_basic.py\ntensorflow_v0.11/2_BasicModels/logistic_regression.py\ntensorflow_v0.11/2_BasicModels/linear_regression.py\ntensorflow_v0.11/2_BasicModels/nearest_neighbor.py\ntensorflow_v0.11/3_NeuralNetworks/bidirectional_rnn.py\ntensorflow_v0.11/3_NeuralNetworks/autoencoder.py\ntensorflow_v0.11/3_NeuralNetworks/recurrent_network.py\ntensorflow_v0.11/3_NeuralNetworks/dynamic_rnn.py\ntensorflow_v0.11/3_NeuralNetworks/multilayer_perceptron.py\ntensorflow_v0.11/3_NeuralNetworks/convolutional_network.py\ntensorflow_v1/1_Introduction/basic_eager_api.py\ntensorflow_v1/1_Introduction/basic_operations.py\ntensorflow_v1/1_Introduction/helloworld.py\ntensorflow_v1/4_Utils/tensorboard_advanced.py\ntensorflow_v1/4_Utils/tensorboard_basic.py\ntensorflow_v1/5_DataManagement/build_an_image_dataset.py\ntensorflow_v1/5_DataManagement/tensorflow_dataset_api.py\ntensorflow_v1/2_BasicModels/random_forest.py\ntensorflow_v1/2_BasicModels/gradient_boosted_decision_tree.py\ntensorflow_v1/2_BasicModels/logistic_regression.py\ntensorflow_v1/2_BasicModels/word2vec.py\ntensorflow_v1/2_BasicModels/linear_regression.py\ntensorflow_v1/2_BasicModels/logistic_regression_eager_api.py\ntensorflow_v1/2_BasicModels/nearest_neighbor.py\ntensorflow_v1/2_BasicModels/linear_regression_eager_api.py\ntensorflow_v1/2_BasicModels/kmeans.py\ntensorflow_v1/6_MultiGPU/multigpu_basics.py\ntensorflow_v1/6_MultiGPU/multigpu_cnn.py\ntensorflow_v1/3_NeuralNetworks/variational_autoencoder.py\ntensorflow_v1/3_NeuralNetworks/neural_network_eager_api.py\ntensorflow_v1/3_NeuralNetworks/neural_network.py\ntensorflow_v1/3_NeuralNetworks/bidirectional_rnn.py\ntensorflow_v1/3_NeuralNetworks/convolutional_network_raw.py\ntensorflow_v1/3_NeuralNetworks/autoencoder.py\ntensorflow_v1/3_NeuralNetworks/gan.py\ntensorflow_v1/3_NeuralNetworks/recurrent_network.py\ntensorflow_v1/3_NeuralNetworks/neural_network_raw.py\ntensorflow_v1/3_NeuralNetworks/multilayer_perceptron.py\ntensorflow_v1/3_NeuralNetworks/convolutional_network.py",
        "cleanup": true,
        "outdir": "/home/vanessa/Desktop/Code/caliper-analysis"
    },
    "tests": {
        "build": {
            "retval": 0
        },
        "tensorflow_v0.11/5_MultiGPU/multigpu_basics.py": {
            "error": [
                "2021-01-05 20:35:15.583939: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA\n",
                "2021-01-05 20:35:15.609662: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1999965000 Hz\n",
                "2021-01-05 20:35:15.610114: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55ff581dd810 executing computations on platform Host. Devices:\n",
                "2021-01-05 20:35:15.610142: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Host, Default Version\n",
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/5_MultiGPU/multigpu_basics.py\", line 61, in <module>\n",
                "    a = tf.placeholder(tf.float32, [1000, 1000])\n",
                "AttributeError: module 'tensorflow' has no attribute 'placeholder'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.26
        },
        "tensorflow_v0.11/1_Introduction/basic_operations.py": {
            "error": [
                "2021-01-05 20:35:18.819434: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA\n",
                "2021-01-05 20:35:18.841672: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1999965000 Hz\n",
                "2021-01-05 20:35:18.842130: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55e305a9fb50 executing computations on platform Host. Devices:\n",
                "2021-01-05 20:35:18.842162: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Host, Default Version\n",
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/1_Introduction/basic_operations.py\", line 25, in <module>\n",
                "    with tf.Session() as sess:\n",
                "AttributeError: module 'tensorflow' has no attribute 'Session'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.91
        },
        "tensorflow_v0.11/1_Introduction/helloworld.py": {
            "error": [
                "2021-01-05 20:35:21.744385: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA\n",
                "2021-01-05 20:35:21.769663: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1999965000 Hz\n",
                "2021-01-05 20:35:21.770133: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x5580c75a3b80 executing computations on platform Host. Devices:\n",
                "2021-01-05 20:35:21.770168: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Host, Default Version\n",
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/1_Introduction/helloworld.py\", line 28, in <module>\n",
                "    sess = tf.Session()\n",
                "AttributeError: module 'tensorflow' has no attribute 'Session'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.96
        },
        "tensorflow_v0.11/4_Utils/tensorboard_advanced.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/4_Utils/tensorboard_advanced.py\", line 21, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.26
        },
        "tensorflow_v0.11/4_Utils/tensorboard_basic.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/4_Utils/tensorboard_basic.py\", line 21, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.97
        },
        "tensorflow_v0.11/2_BasicModels/logistic_regression.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/2_BasicModels/logistic_regression.py\", line 21, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.06
        },
        "tensorflow_v0.11/2_BasicModels/linear_regression.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/2_BasicModels/linear_regression.py\", line 73, in <module>\n",
                "    X = tf.placeholder(\"float\")\n",
                "AttributeError: module 'tensorflow' has no attribute 'placeholder'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.21
        },
        "tensorflow_v0.11/2_BasicModels/nearest_neighbor.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/2_BasicModels/nearest_neighbor.py\", line 23, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.8
        },
        "tensorflow_v0.11/3_NeuralNetworks/bidirectional_rnn.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/3_NeuralNetworks/bidirectional_rnn.py\", line 24, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.04
        },
        "tensorflow_v0.11/3_NeuralNetworks/autoencoder.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/3_NeuralNetworks/autoencoder.py\", line 25, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.0
        },
        "tensorflow_v0.11/3_NeuralNetworks/recurrent_network.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/3_NeuralNetworks/recurrent_network.py\", line 22, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.93
        },
        "tensorflow_v0.11/3_NeuralNetworks/dynamic_rnn.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/3_NeuralNetworks/dynamic_rnn.py\", line 107, in <module>\n",
                "    x = tf.placeholder(\"float\", [None, seq_max_len, 1])\n",
                "AttributeError: module 'tensorflow' has no attribute 'placeholder'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.33
        },
        "tensorflow_v0.11/3_NeuralNetworks/multilayer_perceptron.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/3_NeuralNetworks/multilayer_perceptron.py\", line 13, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.96
        },
        "tensorflow_v0.11/3_NeuralNetworks/convolutional_network.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v0.11/3_NeuralNetworks/convolutional_network.py\", line 21, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.9
        },
        "tensorflow_v1/1_Introduction/basic_eager_api.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/1_Introduction/basic_eager_api.py\", line 23, in <module>\n",
                "    import tensorflow.contrib.eager as tfe\n",
                "ModuleNotFoundError: No module named 'tensorflow.contrib'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.29
        },
        "tensorflow_v1/1_Introduction/basic_operations.py": {
            "error": [
                "2021-01-05 20:36:01.408011: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA\n",
                "2021-01-05 20:36:01.429669: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1999965000 Hz\n",
                "2021-01-05 20:36:01.430157: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55c831232f00 executing computations on platform Host. Devices:\n",
                "2021-01-05 20:36:01.430194: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Host, Default Version\n",
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/1_Introduction/basic_operations.py\", line 25, in <module>\n",
                "    with tf.Session() as sess:\n",
                "AttributeError: module 'tensorflow' has no attribute 'Session'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.89
        },
        "tensorflow_v1/1_Introduction/helloworld.py": {
            "error": [
                "2021-01-05 20:36:04.283508: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA\n",
                "2021-01-05 20:36:04.305657: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1999965000 Hz\n",
                "2021-01-05 20:36:04.306164: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x557423c44870 executing computations on platform Host. Devices:\n",
                "2021-01-05 20:36:04.306196: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Host, Default Version\n",
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/1_Introduction/helloworld.py\", line 28, in <module>\n",
                "    sess = tf.Session()\n",
                "AttributeError: module 'tensorflow' has no attribute 'Session'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.89
        },
        "tensorflow_v1/4_Utils/tensorboard_advanced.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/4_Utils/tensorboard_advanced.py\", line 20, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.27
        },
        "tensorflow_v1/4_Utils/tensorboard_basic.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/4_Utils/tensorboard_basic.py\", line 20, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.84
        },
        "tensorflow_v1/5_DataManagement/build_an_image_dataset.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/5_DataManagement/build_an_image_dataset.py\", line 74, in read_images\n",
                "    classes = sorted(os.walk(dataset_path).next()[1])\n",
                "AttributeError: 'generator' object has no attribute 'next'\n",
                "\n",
                "During handling of the above exception, another exception occurred:\n",
                "\n",
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/5_DataManagement/build_an_image_dataset.py\", line 133, in <module>\n",
                "    X, Y = read_images(DATASET_PATH, MODE, batch_size)\n",
                "  File \"tensorflow_v1/5_DataManagement/build_an_image_dataset.py\", line 76, in read_images\n",
                "    classes = sorted(os.walk(dataset_path).__next__()[1])\n",
                "StopIteration\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.02
        },
        "tensorflow_v1/5_DataManagement/tensorflow_dataset_api.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/5_DataManagement/tensorflow_dataset_api.py\", line 20, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.3
        },
        "tensorflow_v1/2_BasicModels/random_forest.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/random_forest.py\", line 14, in <module>\n",
                "    from tensorflow.contrib.tensor_forest.python import tensor_forest\n",
                "ModuleNotFoundError: No module named 'tensorflow.contrib'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.87
        },
        "tensorflow_v1/2_BasicModels/gradient_boosted_decision_tree.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/gradient_boosted_decision_tree.py\", line 17, in <module>\n",
                "    from tensorflow.contrib.boosted_trees.estimator_batch.estimator import (\n",
                "ModuleNotFoundError: No module named 'tensorflow.contrib'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.13
        },
        "tensorflow_v1/2_BasicModels/logistic_regression.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/logistic_regression.py\", line 21, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.18
        },
        "tensorflow_v1/2_BasicModels/word2vec.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/word2vec.py\", line 58, in <module>\n",
                "    filename, _ = urllib.urlretrieve(url, data_path)\n",
                "AttributeError: module 'urllib' has no attribute 'urlretrieve'\n"
            ],
            "output": [
                "Downloading the dataset... (It may take some time)\n"
            ],
            "retval": 1,
            "seconds": 2.85
        },
        "tensorflow_v1/2_BasicModels/linear_regression.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/linear_regression.py\", line 73, in <module>\n",
                "    X = tf.placeholder(\"float\")\n",
                "AttributeError: module 'tensorflow' has no attribute 'placeholder'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.24
        },
        "tensorflow_v1/2_BasicModels/logistic_regression_eager_api.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/logistic_regression_eager_api.py\", line 20, in <module>\n",
                "    tf.enable_eager_execution()\n",
                "AttributeError: module 'tensorflow' has no attribute 'enable_eager_execution'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.95
        },
        "tensorflow_v1/2_BasicModels/nearest_neighbor.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/nearest_neighbor.py\", line 23, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.92
        },
        "tensorflow_v1/2_BasicModels/linear_regression_eager_api.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/linear_regression_eager_api.py\", line 21, in <module>\n",
                "    tf.enable_eager_execution()\n",
                "AttributeError: module 'tensorflow' has no attribute 'enable_eager_execution'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.21
        },
        "tensorflow_v1/2_BasicModels/kmeans.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/2_BasicModels/kmeans.py\", line 17, in <module>\n",
                "    from tensorflow.contrib.factorization import KMeans\n",
                "ModuleNotFoundError: No module named 'tensorflow.contrib'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.89
        },
        "tensorflow_v1/6_MultiGPU/multigpu_basics.py": {
            "error": [
                "2021-01-05 20:36:46.938360: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA\n",
                "2021-01-05 20:36:46.961620: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 1999965000 Hz\n",
                "2021-01-05 20:36:46.962060: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55bde6aef490 executing computations on platform Host. Devices:\n",
                "2021-01-05 20:36:46.962090: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Host, Default Version\n",
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/6_MultiGPU/multigpu_basics.py\", line 61, in <module>\n",
                "    a = tf.placeholder(tf.float32, [1000, 1000])\n",
                "AttributeError: module 'tensorflow' has no attribute 'placeholder'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.94
        },
        "tensorflow_v1/6_MultiGPU/multigpu_cnn.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/6_MultiGPU/multigpu_cnn.py\", line 28, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.35
        },
        "tensorflow_v1/3_NeuralNetworks/variational_autoencoder.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/variational_autoencoder.py\", line 35, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.93
        },
        "tensorflow_v1/3_NeuralNetworks/neural_network_eager_api.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/neural_network_eager_api.py\", line 26, in <module>\n",
                "    tf.enable_eager_execution()\n",
                "AttributeError: module 'tensorflow' has no attribute 'enable_eager_execution'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.1
        },
        "tensorflow_v1/3_NeuralNetworks/neural_network.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/neural_network.py\", line 20, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.15
        },
        "tensorflow_v1/3_NeuralNetworks/bidirectional_rnn.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/bidirectional_rnn.py\", line 18, in <module>\n",
                "    from tensorflow.contrib import rnn\n",
                "ModuleNotFoundError: No module named 'tensorflow.contrib'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.92
        },
        "tensorflow_v1/3_NeuralNetworks/convolutional_network_raw.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/convolutional_network_raw.py\", line 16, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.13
        },
        "tensorflow_v1/3_NeuralNetworks/autoencoder.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/autoencoder.py\", line 30, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.08
        },
        "tensorflow_v1/3_NeuralNetworks/gan.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/gan.py\", line 35, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.94
        },
        "tensorflow_v1/3_NeuralNetworks/recurrent_network.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/recurrent_network.py\", line 17, in <module>\n",
                "    from tensorflow.contrib import rnn\n",
                "ModuleNotFoundError: No module named 'tensorflow.contrib'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.18
        },
        "tensorflow_v1/3_NeuralNetworks/neural_network_raw.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/neural_network_raw.py\", line 17, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.97
        },
        "tensorflow_v1/3_NeuralNetworks/multilayer_perceptron.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/multilayer_perceptron.py\", line 24, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 2.94
        },
        "tensorflow_v1/3_NeuralNetworks/convolutional_network.py": {
            "error": [
                "Traceback (most recent call last):\n",
                "  File \"tensorflow_v1/3_NeuralNetworks/convolutional_network.py\", line 16, in <module>\n",
                "    from tensorflow.examples.tutorials.mnist import input_data\n",
                "ModuleNotFoundError: No module named 'tensorflow.examples.tutorials'\n"
            ],
            "output": [],
            "retval": 1,
            "seconds": 3.22
        }
    },
    "requirements.txt": [
        "absl-py==0.11.0\n",
        "astor==0.8.1\n",
        "cached-property==1.5.2\n",
        "cachetools==4.2.0\n",
        "certifi==2020.12.5\n",
        "chardet==4.0.0\n",
        "gast==0.2.2\n",
        "google-auth==1.24.0\n",
        "google-auth-oauthlib==0.4.2\n",
        "google-pasta==0.2.0\n",
        "grpcio==1.34.0\n",
        "h5py==3.1.0\n",
        "idna==2.10\n",
        "importlib-metadata==3.3.0\n",
        "Keras-Applications==1.0.8\n",
        "Keras-Preprocessing==1.1.2\n",
        "Markdown==3.3.3\n",
        "numpy==1.19.5\n",
        "oauthlib==3.1.0\n",
        "opt-einsum==3.3.0\n",
        "protobuf==3.14.0\n",
        "pyasn1==0.4.8\n",
        "pyasn1-modules==0.2.8\n",
        "requests==2.25.1\n",
        "requests-oauthlib==1.3.0\n",
        "rsa==4.6\n",
        "six==1.15.0\n",
        "tensorboard==2.0.2\n",
        "tensorflow @ file:///tmp/repo/tensorflow-2.0.2-cp36-cp36m-manylinux2010_x86_64.whl\n",
        "tensorflow-estimator==2.0.1\n",
        "termcolor==1.1.0\n",
        "typing-extensions==3.7.4.3\n",
        "urllib3==1.26.2\n",
        "Werkzeug==1.0.1\n",
        "wrapt==1.12.1\n",
        "zipp==3.4.0\n"
    ]
}